src/XdebugClient.php

```
@@ -499,4 +768,216 @@ public function __destruct()
+            $contents = stream_get_contents($file);
+            if ($contents !== false) {
+                $state = json_decode($contents, true);
+            }
+
+            flock($file, LOCK_UN);
+        }
+
+        fclose($file);
+
+        return $state;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'readStateWithLock'. (Unused Code Rules)

(UnusedPrivateMethod)